doc/releasenotes/16_0_0_summary.md

```diff
@@ -40,6 +40,11 @@ In [PR #11097](https://github.com/vitessio/vitess/pull/11097) we introduced nati
 Orchestrator integration in `vttablet` was deprecated in the previous release and is deleted in this release.
 Consider using `VTOrc` instead of `Orchestrator`.
 
+#### Error Strings
+
+Many errors in the query serving path return different strings. 
+If your application is searching for specific errors, you might need to update this code.
```

Comment:
```suggestion
If your application is searching for specific errors, you might need to update your code.
```

Reply:
resolved in [f2351bb](https://github.com/vitessio/vitess/pull/10738/commits/f2351bba65f347a8e5ce72ff87757ccaf20115af)